gard_owl_ingest/analysis/mondo_mapping_status.py

```diff
@@ -11,41 +15,141 @@
+            mims: List[CURIE] = [f'OMIM:{x}' for x in row.OmimMember[1:-1].split(', ')] if row.OmimMember else []
+            if mims:
+                gard_omim_related[gard_id] = mims
+        # todo: ordo_parents: What about when DataSource == 'GARD'? Will ParentOrphaCode ever exist? Should these marked
+        #  as related instead of broad? Currently, I believe no cases like this though, but should double check.
+        ordo_parents: List[CURIE] = \
+            [f'Orphanet:{x}' for x in row.ParentOrphaCode[1:-1].split(', ')] if row.ParentOrphaCode else []
+        if ordo_parents:
+            gard_ordo_broad[gard_id] = ordo_parents
+
+    # Get explicit, existing mapping status between GARD and Mondo
+    # todo: sort by the `int` of GARD IDs. Account for existence of 1 single GARD ID that is 0-padded
+    existing_df = pd.DataFrame()
+    existing_df['subject_id'] = list(all_ids)
+    existing_df['in_gard'] = existing_df['subject_id'].isin(in_gard)
+    existing_df['in_mondo'] = existing_df['subject_id'].isin(in_mondo)
+    existing_df['status'] = existing_df['subject_id'].apply(
+        lambda x:
+        'gard' if x in in_gard and x not in in_mondo else
+        'mondo' if x in in_mondo and x not in in_gard else
+        'both')
+    existing_df.to_csv(TMP_OUTPUT_DIR / 'gard_terms_mapping_status.csv', index=False)
+
+    # Get proxy mappings between GARD and Mondo, through Orphanet and OMIM
+    rows = []
+    # todo: include: prefix_map etc as comment
+    # todo: include: cols: subject_label, mapping_justification, mondo_label
+    for s, o in gard_ordo_exact.items():
+        rows.append({
+            'subject_id': s,
+            'predicate_id': 'skos:exactMatch',
+            'object_id': o,
+        })
+    for s, o in gard_omim_exact.items():
+        rows.append({
+            'subject_id': s,
+            'predicate_id': 'skos:exactMatch',
+            'object_id': o,
+        })
+    for s, o_list in gard_ordo_broad.items():
+        for o in o_list:
+            rows.append({
+                'subject_id': s,
+                'predicate_id': 'skos:broadMatch',
+                'object_id': o,
+            })
+    for s, o_list in gard_omim_narrow.items():
+        for o in o_list:
+            rows.append({
+                'subject_id': s,
+                'predicate_id': 'skos:narrowMatch',
+                'object_id': o,
+            })
+    for s, o_list in gard_omim_related.items():
+        for o in o_list:
+            rows.append({
+                'subject_id': s,
+                'predicate_id': 'skos:relatedMatch',
+                'object_id': o,
+            })
+    gard_sssom_df = pd.DataFrame(rows)
+    gard_sssom_df2 = pd.merge(
+        gard_sssom_df, mondo_sssom_df, how='left', left_on='object_id', right_on='mondo_object_id').fillna('')\
+        .sort_values(by='subject_id')
+    del gard_sssom_df2['mondo_object_id']
+    gard_sssom_df2.to_csv(TMP_OUTPUT_DIR / 'gard.sssom.tsv', index=False, sep='\t')
```

Comment:
FYI: New output, `gard.sssom.tsv`. Though it also has two non-sssom columns: `mondo_id` and `mondo_label`